src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -72,12 +72,12 @@ class CLIJsonWebKeyOperation(str, Enum):
         c.argument('spn', help='name of a service principal that will receive permissions')
         c.argument('upn', help='name of a user principal that will receive permissions')
         c.argument('tags', tags_type)
-        c.argument('enabled_for_deployment', arg_type=get_three_state_flag(), help='Allow Virtual Machines to retrieve certificates stored as secrets from the vault.')
-        c.argument('enabled_for_disk_encryption', arg_type=get_three_state_flag(), help='Allow Disk Encryption to retrieve secrets from the vault and unwrap keys.')
-        c.argument('enabled_for_template_deployment', arg_type=get_three_state_flag(), help='Allow Resource Manager to retrieve secrets from the vault.')
-        c.argument('enable_rbac_authorization', arg_type=get_three_state_flag(), help='Enable RBAC authorization for the vault, and all contained entities.')
-        c.argument('enable_soft_delete', arg_type=get_three_state_flag(), help='Enable vault deletion recovery for the vault, and all contained entities')
-        c.argument('enable_purge_protection', arg_type=get_three_state_flag(), help='Prevents manual purging of deleted vault, and all contained entities')
+        c.argument('enabled_for_deployment', arg_type=get_three_state_flag())
```

Comment:
why remove the help message?

Reply:
> why remove the help message?

@yungezz By removing the help message here, we can use help messages from Swagger directly (this would meet KeyVault team's requirement).